open_schools_platform/query_management/queries/filters.py

```diff
@@ -1,8 +1,22 @@
+from django.db.models import Q
+from django_filters import CharFilter
+
 from open_schools_platform.common.filters import BaseFilterSet
 from open_schools_platform.query_management.queries.models import Query
 
 
 class QueryFilter(BaseFilterSet):
+    sender_ct_search = CharFilter(field_name="sender_ct", method="sender_ct_filter")
+    recipient_ct_search = CharFilter(field_name="recipient_ct", method="recipient_ct_filter")
+
+    def sender_ct_filter(self, queryset, name, value):
+        return queryset.filter(Q(sender_ct__app_label__icontains=value) |
+                               Q(sender_ct__model__icontains=value.replace(" ", "")))
+
+    def recipient_ct_filter(self, queryset, name, value):
+        return queryset.filter(Q(sender_ct__app_label__icontains=value) |
+                               Q(recipient_ct__model__icontains=value.replace(" ", "")))
```

Comment:
are this filters correct?
why we also search in app_label?

Reply:
Since the content type that is stored in the Query (as body_ct and recipient_ct) model has two parts ("app_label" | "model"), I thought it would be better to search on both of them.

Reply:
but how app_labels are related to model names?
we want to filter by entity name, not by apps
actually, should we use icontains there?

Reply:
Actually yeah, I agree with u, searching by app_label is unnecessary here.
And about icontains, it can be convenient sometimes, but as I think about it, it can be confusing as well. (for example, we want to find query with sender = student_profile, we type down "student" in filter and it will find correct queries, but student is also body for this kind of queries) 

So I am going to remove search by app_label and icontains parameter.